ci/before-install.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# shellcheck source=lib/stdlib.bash
+source "$(dirname "${BASH_SOURCE[0]}")/lib/stdlib.bash" || exit 67
+
+os_name="$(go env GOOS)"
+
+if [[ "${os_name}" == "darwin" ]]; then
+  # bash from macOS is too old to have readarray. Install newer version.
+  brew install bash
+elif [[ "${os_name}" == "linux" ]]; then
+  echo "Nothing to do for linux."
```

Comment:
I noticed the docker-version test inside seccomp_options gives output that looks like an error, which lead me to see that the version of docker by default in travis is truly ancient, so I thought I'd leave space in here to install it, and so I extracted from .travis.yml to here.